src/command_modules/azure-cli-network/azure/cli/command_modules/network/mgmt_route_table/azuredeploy.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "routeTableName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name for route table."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for resource."
```

Comment:
>resource. [](start = 37, length = 9)

Do we need to mention it defaults to the resource group location? I don't think it could populate in help automatically.